### ISSUE TITLE
Why N is not defined as a parameter when we define a new model ?

### ISSUE THREAD

**Issue by SergeStinckwich** [UMMISCO/kendrick#99]
Why do we distinguish N from other parameters when we define a model ?

Example : 
```Smalltalk
model := KEModel new.
	model population: (KEPopulation size: 5000).
	SEIRConcern := KEModelPart new.
	SEIRConcern attributes: {(#status -> #(#S #E #I #R))}.
	SEIRConcern addParameters: {#beta . #gamma . #mu . #sigma}.
	SEIRConcern addParameter: #lambda value: 'beta*I/N'.
	SEIRConcern
		addEquations:
			{'S:t=mu*N - lambda*S - mu*S' . 'E:t=lambda*S - sigma*E - mu*E' . 'I:t=sigma*E - gamma*I - mu*I' . 'R:t=gamma*I - mu*R'}.
```
N is not defined as a parameter like beta, gamma, ...

**Comment by mikalziane** [comment]
N is an implicit parameter whose value is Card(P) where P is the whole population

**Comment by SergeStinckwich** [comment]
Yes this is an implicit parameter, but at the moment there is a lot of code that deals with N. IMHO, if if N is defined as a parameter automatically added everytime we build a new model, this will simplify the code.

**Comment by mikalziane** [comment]
Yes I agree: the fact that N is a parameter, even implicit, should not be in ST code. Maybe it should be added automatically when translating the K DSL to lower-level K? This would be in one place only ideally.

**Comment by SergeStinckwich** [comment]
At the moment, if you inspect a model, N does not appears in the parameters of a model. So N appear in the code, but not the model :-(

**Comment by mikalziane** [comment]
The fact that N is not in the Kendrick DSL text is ok I think. Now inspected in ST is dfferent. It should appear!

**Comment by btmanh** [comment]
I think N has to appear in the parameters of model, as in the initialize function of model, N and t is automatically added as parameters:
initialize
	super initialize.
	t := 0.
	self addParameter: #N value: [ :aModel | aModel population size ].
	self addParameter: #t value: [ :aModel | aModel t ].
	...

**Comment by SergeStinckwich** [comment]
Ok but then propose a PR to fix this issue.

**Comment by YvanGuifo** [comment]
I don't understand why we should explicitly define N

**Comment by mikalziane** [comment]
Any parameter should be defined somewhere even if the definition is implicit in the high level DSL it is a best practice I think to make assumptions explicit at the lower level, otherwise it will be hard coded, more difficult to understand and change.
